Packages/com.unity.inputsystem/InputSystem/InputSettings.cs

```diff
@@ -702,6 +702,27 @@ public bool shortcutKeysConsumeInput
             }
         }
 
+
+        /// <summary>
+        /// Enables or disables logging of amount of processed input events for each device. This is done per
+        /// InputSystem.Update() call.
+        /// </summary>
+        ///
+        /// This should only be used for debugging purposes. It is useful to track down devices that are sending
+        /// a lot of input events per frame. It will pollute the console log with a lot of messages.
+        public bool debugLogProcessedEventForDevice
```

Comment:
I do not think such a setting belong in InputSettings since InputSettings are shared among developers on a project. E.g. if you turn on debugging I wouldn't expect to get debugging turned on if I checkout the same repo? I would instead suggest that this is saved under Preferences, e.g. https://docs.unity3d.com/ScriptReference/EditorPrefs.html or what do you think?

Reply:
Good point, it wouldn't make sense to use InputSettings then.